todocore/src/main/java/com/knighttodo/todocore/service/DayTodoService.java

```diff
@@ -75,8 +76,8 @@ public void deleteById(UUID dayTodoId) {
         dayTodoRepository.softDeleteById(dayTodoId);
     }
 
-    public List<DayTodoVO> findByDayId(UUID dayId) {
-        return dayTodoRepository.findByDayIdAlive(dayId).stream().map(dayTodoMapper::toTodoVO)
+    public List<DayTodoVO> findByDayOrderNumber(UUID dayId) {
+        return dayTodoRepository.findAllWithOrderNumberAlive(dayId).stream().map(dayTodoMapper::toTodoVO)
```

Comment:
You shouldn't rename this method we still looking by DayId

Reply:
Done!